packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -88,6 +272,7 @@ export class GaussianSplattingMesh extends Mesh {
         this._material = value;
         this._material.backFaceCulling = true;
         this._material.cullBackFaces = false;
+        this.subMeshes[0].materialDefines = null;

```

Comment:
You should not have to do this, I will have a look tomorrow.

Reply:
this is the webgl1 failing PG used for validation tests: https://playground.babylonjs.com/#WXWJKN#4

Reply:
This would be the link to test with - https://playground.babylonjs.com/?snapshot=refs/pull/15734/merge#WXWJKN#4

WebGL1 tests are running virtualized on an ubuntu, so it is always possible the environment is causing this issue. I can't reproduce this on a mac on both chrome and safari